bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/CameraCapability.java

```diff
@@ -62,12 +63,15 @@ public CameraCapability(CommonInterface handler, NetatmoDescriptionProvider desc
     @Override
     public void updateHomeStatusModule(HomeStatusModule newData) {
         super.updateHomeStatusModule(newData);
-        String vpnUrl = newData.getVpnUrl();
-        if (vpnUrl != null) {
-            localUrl = newData.isLocal() ? securityCapability.map(cap -> cap.ping(vpnUrl)).orElse(null) : null;
-            cameraHelper.setUrls(vpnUrl, localUrl);
-            eventHelper.setUrls(vpnUrl, localUrl);
+        // Per documentation vpn_url expires every 3 hours and on camera reboot. So useless to reping it if not changed
```

Comment:
But the ping was to detect an unplugged camera. Unplugged and rebooted is very different.
Did you check what happens to vpn_url when you just unplug your camera ?

Reply:
Is your idea to wait until 3 hours to detect an unplugged camera ?

Reply:
Not exactly. The ping is only done to get the localUrl. Without it you can not take local actions toward the Camera. An unplugged camera will be reported by the module being inactive, so status changed to OFFLINE.

Reply:
Ok, fine, I understand.
Did you check after what delay the camera thing switches to OFFLINE with your changes after you unplug it? And ONLINE after you plug it again?

Reply:
No, this is not changed at all by my current changes. Being inactive, or unreachable is reported by Netatmo API, at best it will be reported on the next recurrent poll of the API. How, how often and when Netatmo API detects a camera as being offline I'be never measured it (at home, I use the FreeboxOS network devices report to monitor this).

Reply:
If you confirm there is no change regarding detection of camera reachability, that is fine for me.

Reply:
I do :)